src/User/Concerns/DeprecatedUserNotificationPreferences.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+/*
+ * This file is part of Flarum.
+ *
+ * For detailed copyright and license information, please view the
+ * LICENSE file that was distributed with this source code.
+ */
+
+namespace Flarum\User\Concerns;
+
+trait DeprecatedUserNotificationPreferences
+{
+    /**
+     * Encode an array of preferences for storage in the database.
+     *
+     * @param mixed $value
+     * @deprecated 0.1.0-beta.13: `users.preferences` is no longer used.
+     */
+    public function setPreferencesAttribute($value)
```

Comment:
If they're being deprecated, shouldn't the methods in this class still contain an implementation so they don't break dependent code until we remove them in beta 14?